src/login/logind.c

```diff
@@ -63,6 +63,7 @@ static void manager_reset_config(Manager *m) {
 
         m->runtime_dir_size = PAGE_ALIGN((size_t) (physical_memory() / 10)); /* 10% */
         m->user_tasks_max = UINT64_C(12288);
+        m->sessions_max = UINT64_C(8192);
```

Comment:
humm. good question. does "but but the line before did it the same way" count as an excuse? ;-)

will drop this.